### PR TITLE
Fix SerializedName and name converter

### PR DESCRIPTION
**Pull request by Zowac**
| Q             | A
| ------------- | ---
| Branch?       | 3.1
| Tickets       | #4752 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

**Comment by soyuka** [comment]
tests are failing?

**Comment by Zowac** [comment]
> tests are failing?

It's fixed :)

File: src/Serializer/AbstractItemNormalizer.php [hunk 1]
```diff
@@ -222,8 +222,16 @@ public function denormalize(mixed $data, string $class, string $format = null, a
...
         // Revert attributes that aren't allowed to be changed after a post-denormalize check
         foreach (array_keys($data) as $attribute) {
+            $attribute = $this->nameConverter ? $this->nameConverter->denormalize((string) $attribute) : $attribute;
+            if (!\in_array($attribute, $propertyNames, true)) {
+                continue;
+            }
+
             if (!$this->canAccessAttributePostDenormalize($object, $previousObject, $attribute, $context)) {
                 if (null !== $previousObject) {
                     $this->setValue($object, $attribute, $this->propertyAccessor->getValue($previousObject, $attribute));
```
Comment (soyuka) [inline src/Serializer/AbstractItemNormalizer.php:237]: There's a change missing line 239 you can reuse the same options.

Comment (Zowac) [reply src/Serializer/AbstractItemNormalizer.php:237]: Fixed, thanks :)

**Comment by soyuka** [comment]
thanks!

**Comment by michaljusiega** [comment]
Can we backport this fix into 2.7 ?

**Comment by soyuka** [comment]
please do I'll merge it

**Pull request merged**